build.gradle

```diff
@@ -231,6 +231,19 @@ project(':iceberg-core') {
       exclude group: 'org.slf4j', module: 'slf4j-log4j12'
     }
   }
+
+  task testJar(type: Jar){
+    archiveClassifier = 'tests'
+    from sourceSets.test.output
+  }
+
+  configurations {
```

Comment:
We've define `testArtifacts` configuration in `iceberg-hive` as well. I think it makes sense to define in under `allprojects` or `subprojects` once.  Same for `testJar`